recipe/meta.yaml

```
@@ -1,29 +1,30 @@
 {% set name = "opentelemetry-api" %}
-{% set version = "1.30.0" %}
+{% set version = "1.37.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentelemetry_api-{{ version }}.tar.gz
-  sha256: 375893400c1435bf623f7dfb3bcd44825fe6b56c34d0667c542ea8257b1a1240
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: 540735b120355bd5112738ea53621f8d5edb35ebcd6fe21ada3ab1c61d1cd9a7
 
 build:
   number: 0
-  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<39]
 
 requirements:
   host:
     - pip
     - python
     - hatchling
   run:
-    - deprecated >=1.2.6
-    - importlib-metadata >=6.0,<=8.5.0
     - python
+    - typing-extensions >=4.5.0
+    - importlib-metadata >=6.0,<=8.5.0
```

Recommend testing without this on above 3.12 - at the very least we should bump up the version for all python versions

https://github.com/open-telemetry/opentelemetry-python/blob/5c8269c2d33617cca26c63e8627aec89e0d79b17/opentelemetry-api/pyproject.toml#L29


```suggestion
    - importlib-metadata >=6.0,<8.8.0 # [py<312]
```

---

thanks

---

https://taskcluster-prod.s3.us-east-1.amazonaws.com/a5bJwFhZREKOiEtDdhnSRA/0/public/logs/live_backing.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAWUI46DZFCZGHCOQD%2F20250919%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250919T085847Z&X-Amz-Expires=1800&X-Amz-Signature=b14984b1b31d64ea3d7bb28b8fb4c7ac18ed512fc6aded00e4b54245d29474d1&X-Amz-SignedHeaders=host&x-id=GetObject

```
import: 'opentelemetry'
import: 'opentelemetry.context'
Traceback (most recent call last):
  File "/opt/miniconda3/conda-bld/opentelemetry-api_1758272245523/test_tmp/run_test.py", line 5, in <module>
    import opentelemetry.context
  File "/opt/miniconda3/conda-bld/opentelemetry-api_1758272245523/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.12/site-packages/opentelemetry/context/__init__.py", line 26, in <module>
    from opentelemetry.util._importlib_metadata import entry_points
  File "/opt/miniconda3/conda-bld/opentelemetry-api_1758272245523/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.12/site-packages/opentelemetry/util/_importlib_metadata.py", line 20, in <module>
    from importlib_metadata import (  # type: ignore
ModuleNotFoundError: No module named 'importlib_metadata'
```
The selector must be removed.